pycytominer/tests/test_cyto_utils/test_feature_drop_outlier.py

```diff
@@ -23,29 +23,34 @@
         "Cells_x": [1, 2, -8, 2, 5, 5, 5, -1],
         "Cytoplasm_y": [3, -1, 7, 4, 5, -9, 6, 1],
         "Nuclei_z": [-1, 8, 2, 5, -6, 20, 2, -2],
-        "Cells_zz": [14, -46, 1, 60, -30, -100, 2, 2],
+        "Cells_zz": [14, -46, 1, 60, -30, -10000, 2, 2],
     }
 ).reset_index(drop=True)
 
 
 def test_outlier_default():
     result = drop_outlier_features(data_df)
-    expected_result = ["Cells_zz", "Nuclei_z"]
+    expected_result = ["Cells_zz"]
     assert sorted(result) == sorted(expected_result)
 
 
-def test_outlier_high_cutoff():
+def test_outlier_30_cutoff():
```

Comment:
This test and `test_outlier_default()` seem essentially the same to me. We can probably delete this one, especially given the addition of `test_outlier_15_cutoff()`